packages/algob/src/lib/tx.ts

```diff
@@ -231,3 +231,24 @@ export async function executeTransaction (
     return confirmedTx;
   }
 }
+
+/**
+ * Description: decode signed txn from file and send to network.
+ * probably won't work, because transaction contains fields like
+ * firstValid and lastValid which might not be equal to the
+ * current network's blockchain block height.
+ * @param deployer AlgobDeployer
+ * @param fileName raw signed txn .tx file
+ */
+export async function executeSignedTxnFromFile (
+  deployer: AlgobDeployer,
+  fileName: string): Promise<tx.ConfirmedTxInfo> {
+  const signedTxn = loadSignedTxnFromFile(fileName);
+  if (signedTxn === undefined) {
+    throw new Error(`File ${fileName} does not exist`);
+  }
+  console.log("Decoded txn from %s: %O", fileName, decodeSignedTransaction(signedTxn));
```

Comment:
do we need this log? Maybe we should use `console.debug` here?

Reply:
Yes, `console.debug` would be better. Just wanted to show the use of `decodeSignedTransaction(..)` here.